newrelic/docs-website
# Add check-image-import script to verify-mdx

### ✨ Summary

- refactor image script for readability and use as a util
- delete old script and yarn alias
- remove noisy terminal output from verify-mdx
- replace output with terminal progress bar
- refactor mdx output to not interrupt progress bar
- add formatting to terminal outputs for better readability

**Gotcha**: There are currently (26*) existing image import errors throughout the repo if you run `verify-mdx` on all content files. We will create a follow up ticket to fix them and communicate with tech writers about it.

*Previously about 100, but the simple whitespace errors where addressed [here](https://github.com/newrelic/docs-website/pull/15116)

### QA:
Create mdx or image import errors in an mdx file and run `yarn verify-mdx` or `yarn verify-mdx <filePath>` to see it run 

### 🖼️ Screenshots
<img width="1198" alt="Screenshot 2023-11-07 at 8 58 21 AM" src="https://github.com/newrelic/docs-website/assets/47728020/3c00cd7f-00d9-4af3-a1ca-028dc914733b">
<img width="1198" alt="Screenshot 2023-11-06 at 5 40 07 PM" src="https://github.com/newrelic/docs-website/assets/47728020/13a602f6-7e4f-488f-859c-3a026aff4138">

### 🔮 Engineering checklist



- [x] Relevant documentation including README's or Confluence are updated to reflect these changes
- [x] Code changes are self-documenting or clearly commented/documented
- [ ] Changes that effect tech writers have been communicated to tech docs team
